server/service/apple_mdm.go

```diff
@@ -810,7 +810,7 @@ var enrollmentProfileMobileconfigTemplate = template.Must(template.New("").Parse
 	<key>PayloadDisplayName</key>
 	<string>{{ .Organization }} Enrollment</string>
 	<key>PayloadIdentifier</key>
-	<string>com.fleetdm.fleet.mdm.apple</string>
+	<string>` + apple_mdm.FleetPayloadIdentifier + `</string>
```

Comment:
What do you mean? 

`apple_mdm.FleetPayloadIdentifier` is a constant because we are using it both as the PayloadIdentifier in the MDM enroll profile, AND we are using it to identify Fleet MDM.

Reply:
lol, my first message makes no sense, sorry about that. I meant why not:

```suggestion
	<string>{{ .FleetPayload }}</string>
```

and send the value to the template, but it's really not important